jung-visualization/pom.xml

```diff
@@ -9,9 +9,33 @@
   <artifactId>jung-visualization</artifactId>
   <name>JUNG - Visualization Support</name>
   <description>Core visualization support for the JUNG project</description>
-
+  <properties>
+    <version.commons-logging>1.2</version.commons-logging>
+    <version.log4j>1.2.16</version.log4j>
```

Comment:
If we really do want to keep log4j (which I admit I'm sceptical about currently), please consider using log4j 2.